src/core/server/saved_objects/migrationsv2/actions/create_index.test.ts

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { catchRetryableEsClientErrors } from './catch_retryable_es_client_errors';
```

Comment:
the current PR shows that we don't have good test coverage in the unit tests. we also have some integration tests for actions https://github.com/elastic/kibana/blob/5ff9382330d38763703fd05e308742495190c480/src/core/server/saved_objects/migrationsv2/integration_tests/actions.test.ts

Since we are not splitting the tests by action type (as this would significantly slow down their execution) should we at least move them closer to this `/actions` folder?